app/models/concerns/orchestration.rb

```diff
@@ -22,6 +22,14 @@ def rebuild_methods=(methods)
       @rebuild_methods = methods || {}
     end
 
+    def rebuild_methods_for(only = nil)
+      if only.present?
+        (@rebuild_methods || {}).select { |k,v| only.include?(v) }
+      else
+        @rebuild_methods || {}
+      end
+    end
```

Comment:
Hey sorry for the delay, tests are red please rebase and this can be rewritten as:

    def rebuild_methods_for(only = valid_rebuild_only_values)
      (@rebuild_methods || {}).select { |k,v| only.include?(v) }
    end

Or similar (not sure if the cop will like that).